utils/techutils/techutils.go

```diff
@@ -44,10 +44,11 @@ const (
 	Cocoapods Technology = "cocoapods"
 	Swift     Technology = "swift"
 	NoTech    Technology = ""
+	Gem       Technology = "gem"
```

Comment:
Please add Ruby to `CodeLanguage` for it as well
add it to `packageTypes` below.